### ISSUE TITLE
Time steps in days?

### ISSUE THREAD

**Issue by milinda-abay** [jtrauer/summer_python#3]
https://github.com/jtrauer/summer_python/blob/0c4622ae26a4fab0800ce920c6c805b9a776f81b/tb_model.py#L97

Instead of using the format 'year.decimal' is it possible to use integer days as a time step?

PowerBI has a date/time format with baked-in time intelligence functions. 
Running epi models in integer days will facilitate the conversion to PowerBI date/time. It might even improve model runtimes. ie. integer vs floats.

**Comment by jtrauer** [comment]
Yep, great idea. Sometimes we find that it is faster to integrate with whatever time-step is fastest and then interpolate to get our results - perhaps that's what the ODE solver is doing anyway, not sure. Alternatively, could we perhaps define the times as year decimals for the ODE solver and then just convert back to date format after integration...?
